src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,231 @@
+                "provided if radius is given."
+            )
+        separations = stars["ra_dec"].separation(pointing)
+        stars["separation"] = separations
+        stars = stars[separations < radius]
+
+    return stars
+
+
+def get_star_catalog(catalog, filename):
+    """
+    Utility function to download a star catalog for the get_bright_stars function
+
+    Parameters
+    ----------
+    catalog: string
+        Name of the catalog to be used. Usable names are found in the Enum StarCatalogues. Default: Yale
+    filename: string
+        Name and location of the catalog file to be produced
+    """
+    from astroquery.vizier import Vizier
+
+    vizier = Vizier(
+        catalog=StarCatalogues[catalog].value,
+        columns=[
+            "recno",
+            "RAJ2000",
+            "DEJ2000",
+            "RAICRS",
+            "DEICRS",
+            "pmRA",
+            "pmDE",
+            "Vmag",
+            "Bmag",
+            "BTmag",
+            "VTmag",
+        ],
+        row_limit=1000000,
+    )
+
+    stars = vizier.query_constraints(Vmag="0.0..10.0")[0]
+
+    if "Bmag" not in stars.keys():
+        log.exception("The chosen catalog does not have Bmag data")
+    if "Vmag" not in stars.keys():
+        log.exception("The chosen catalog does not have Vmag data")
+    stars.meta["Catalog"] = StarCatalogues[catalog].value
+
+    stars.write(CACHE_FILE, overwrite=True)
+
+
+def get_bright_stars(
+    pointing=None, radius=None, Bmag_cut=None, Vmag_cut=None, catalog="Yale"
+):  # max_magnitude):
+    """
+    Get an astropy table of bright stars from a VizieR catalog
+
+    You can browse the catalogs at https://vizier.cds.unistra.fr/viz-bin/VizieR
 
     Parameters
     ----------
     pointing: astropy Skycoord
        pointing direction in the sky (if none is given, full sky is returned)
     radius: astropy angular units
        Radius of the sky region around pointing position. Default: full sky
-    magnitude_cut: float
-        Return only stars above a given magnitude. Default: None (all entries)
+    Vmag_cut: float
+        Return only stars above a given apparent magnitude. Default: None (all entries)
+    Bmag_cut: float
+        Return only stars above a given absolute magnitude. Default: None (all entries)
+    catalog: string
+        Name of the catalog to be used. Usable names are found in the Enum StarCatalogues. Default: Yale
 
     Returns
     -------
     Astropy table:
        List of all stars after cuts with names, catalog numbers, magnitudes,
        and coordinates
     """
-    from ctapipe.utils import get_table_dataset
 
-    catalog = get_table_dataset("yale_bright_star_catalog5", role="bright star catalog")
+    stars = None
 
-    starpositions = SkyCoord(
-        ra=Angle(catalog["RAJ2000"], unit=u.deg),
-        dec=Angle(catalog["DEJ2000"], unit=u.deg),
-        frame="icrs",
-        copy=False,
-    )
-    catalog["ra_dec"] = starpositions
+    if CACHE_FILE.exists():
```

Comment:
This should use the importlib resources system to load the resources file for the specified catalog